### ISSUE TITLE
Long Polling

### ISSUE THREAD

**Issue by Ansula1267** [robertklep/bosch-xmpp#45]
If I would like to create a history of the state/values of a thermostat I can of course request the values of the relevant endpoints regularly, e.g every minute. Not very elegant, as most of the time nothing will change. For the Bosch Smart Home Controller there is a mechanism to subscribe to changes and get notified: the "Long Polling". Is something similar available from bosch-xmpp (in bridge mode)?

**Comment by robertklep** [comment]
No, there's nothing similar available, but I never really looked at the possibility for an event-driven configuration of the client.

For now, regular polling is the only way I'm afraid.
